index.js

```diff
@@ -11,35 +11,40 @@ function TaserReporter(baseReporterDecorator, config) {
     // This is not a full list of events that can be listened for, only the set we currently care about
     this.onRunStart = onRunStart.bind(this);
     this.onBrowserStart = onBrowserStart.bind(this);
-    this.onBrowserError = onBrowserError.bind(this);
```

Comment:
Why was onBrowserStart and onBrowserError merged? Now they have if conditions and it makes the code tougher to understand.